.pre-commit-config.yaml

```diff
@@ -0,0 +1,9 @@
+repos:
```

Comment:
Maybe we should include some notes in the README on dev environment setup? Otherwise contributors may not configure the pre commit hooks properly. I for one have never actually used the pre commit plugin.

Reply:
My bad I will leave this for a separate PR dealing qith balck and all those gooides